src/renderers/react/ElementWrapper.tsx

```diff
@@ -39,26 +42,19 @@ const Panel = styled("div")`
   padding-right: ${space[3]}px;
 `;
 
-const Actions = styled("div")`
-  display: flex;
-  flex-direction: column;
-  opacity: 0;
-  transition: opacity 0.2s;
-`;
-
-const Button = styled("button")`
+const Button = styled("button")<{ expanded?: boolean }>`
   appearance: none;
   background: ${neutral[93]};
   border: none;
   border-top: 1px solid ${neutral[100]};
   color: ${neutral[100]};
   cursor: pointer;
-  flex-grow: ${({ expanded }: { expanded?: boolean }) =>
-    expanded ? "1" : "0"};
+  flex-grow: ${({ expanded }) => (expanded ? "1" : "0")};
```

Comment:
I may be missing something obvious, but what is this expanded used for / come from? When is it ever defined?

Reply:
If the expanded prop exists on the styled component, e.g:
<img src="https://user-images.githubusercontent.com/34686302/131508721-60b45c15-925a-49f2-8185-18b7eacda001.png" width="300px"></img>